packages/email-field/src/vaadin-email-field.js

```diff
@@ -49,6 +49,15 @@ export class EmailField extends TextField {
     return 'vaadin-email-field';
   }
 
+  static get properties() {
+    return {
+      pattern: {
+        type: String,
+        value: '^([a-zA-Z0-9_\\.\\-+])+@[a-zA-Z0-9-.]+\\.[a-zA-Z0-9-]{2,}$',
```

Comment:
This could be simplified by moving this to the `constructor()`:

```js
this.pattern = '^([a-zA-Z0-9_\\.\\-+])+@[a-zA-Z0-9-.]+\\.[a-zA-Z0-9-]{2,}$';
```

This makes the `static get properties()` block unnecessary.

Reply:
Indeed, let me do this.

Reply:
Done.